contrib/cray/Jenkinsfile.verbs

```diff
@@ -50,6 +50,12 @@ pipeline {
                 sh "./configure --prefix=$LIBFABRIC_INSTALL"
                 sh "make -j 12"
                 sh "make install"
+                dir ("fabtests") {
+                    sh './autogen.sh'
+                    sh "./configure --with-libfabric=$LIBFABRIC_INSTALL --prefix=$FABTEST_PATH"
+                    sh "make -j12"
+                    sh "make -j12 install"
```

Comment:
Why do you have two "makes -j 12"?

Reply:
make -j12 only does the build portion.
make -j12 install installs the files if all of the files are prebuilt.